examples/README.md

```diff
@@ -0,0 +1,65 @@
+## Optimizely Agent Examples
+
+This folder provides a set of sample scripts to illustrate some of the main API features.
+
+### Installation
+
+The included examples were written in python and assumes python 3.7+. If using a python3 venv [virtual environment](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/)
+you can install all of the dependencies in the requirement.txt:
+```bash
+pip install -r requirements.txt
+```
+
+### Activate
+
+The `/activate` endpoint returns a decision of the requested experiment or feature for a given user context.
+For single decisions please refer to [basic.py](./basic.py) which demonstrates how to iterate through the configuration for
+a given SDK project and make individual activation requests.
+
+Example usage:
+```bash
+python basic.py <SDK-Key>
```

Comment:
Nice, maybe make one note somewhere in this document that you can find your SDK Key in the settings of your Optimizely Full Stack or Rollouts account.

Reply:
I just added a mention at the top of the README.md